MEAnalysis/python/MakeDatasetFiles.py

```diff
@@ -166,7 +166,7 @@
             #merge lumi files
 
         #merge lumi files
-        total_lumis = lumis[1]
+        total_lumis = LumiList()
```

Comment:
OK, it should have been `lumis[0]`. I don't think it double counts though, because you are doing an OR. But they way you did it is fine as well, because you do an OR with an empty set.